index.html

```diff
@@ -1239,12 +1330,11 @@
 );
 </pre>
       <h3>
-        WebGL Fragment Shader based post-processing
+        WebGL2: <dfn>upload to float texture</dfn>
       </h3>
       <pre class="example">
-// This code sets up a video element from a depth stream, uploads it to a WebGL
-// texture, and samples that texture in the fragment shader, reconstructing the
-// 16-bit depth values from the red and green channels.
+// This code sets up a video element from a depth stream, uploads it to a WebGL2
+// float texture containing the full precision data as the 16-bit depth map.
```

Comment:
removed the comment.
depth map is 16 bit. It is not exposed.
The texture is float.